File: cpp/27_Remove_Element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int i = 0;
        int j = 0;
        for(i = 0; i < nums.size(); i++){
            if(nums[i] == val){
                continue;
            }else if(nums[j] == val){
                swap(nums[i],nums[j]);
                j++;
            }else{
                j++;
            }
             
        }
        return j;
    }
};

File: 27_Remove_Element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        left,right = 0, len(nums)-1
        while left <= right:
            if nums[right] == val:
                right -= 1
            elif nums[left] == val:
                nums[right],nums[left] = nums[left],nums[right] 
                right -= 1
            else:
                left += 1
        return right + 1